nasa-gibs/worldview
# WV-2887 Preload Tour Story Imagery (no custom palettes)

## Description
This adds preloading to all the imagery in a tour story when a tour story is initiated. Layers that have custom palettes, custom min/max, and disabled values are not preloaded.

## How To Test
1. `git checkout wv-2887-preload-tourstory-noncustom`
2. `npm ci`
3. `npm run watch`
4. Open any Tour Story and navigate through each step.
5. Verify that the imagery for each step is already preloaded before being the step is rendered onscreen.
